hare/messagevalidation.go

```diff
@@ -10,12 +11,13 @@ type messageValidator interface {
 }
 
 type eligibilityValidator struct {
-	oracle HareRolacle
+	oracle Rolacle
+	n      int
 	log.Log
 }
 
-func NewEligibilityValidator(oracle HareRolacle, logger log.Log) *eligibilityValidator {
-	return &eligibilityValidator{oracle, logger}
+func NewEligibilityValidator(oracle Rolacle, n int, logger log.Log) *eligibilityValidator {
```

Comment:
Add a comment explaining what is `n`. 
GENERAL COMMENT FOR THE ENTIRE PR-
Add documentation to ALL exported methods.

Reply:
Also, you should also pass the size of the leader committee. Currently in `expectedCommitteeSize` (in mock_oracle.go) it is hard-coded to 1. This number should be 5.

Reply:
Added for n.
THEY ARE ALL DOCUMENTED (except New since it is clear what new does, unless there are multiple newS).

Reply:
Regarding the leader comm size - should it be in the config?

Reply:
If you decide not to document constructors then you must make sure that all params have meaningful names

Reply:
Yes, the size of the leader committee should be in the configuration

Reply:
Done